script/build-app.sh

```diff
@@ -52,9 +52,10 @@ BUILDS=builds
 RELEASE="$NAME-$VERSION-$OS"
 RELEASE_DIR="$BUILDS/$RELEASE"
 RELEASE_TARBALL="$BUILDS/${RELEASE}.tar.gz"
+RELEASE_ZIP="${RELEASE}.zip"
```

Comment:
I can definitely use `pushd $BUILDS` but if I change this line to `RELEASE_ZIP="$BUILDS/${RELEASE}.zip"` then the zip file is created with the files nested under e.g. *builds\lighttable-0.8.0-windows* instead of *lighttable-0.8.0-windows* and the former is ugly (tho possibly consistent with the tarball).